### PR TITLE
Added in WETHx downgrade widget, cleaned up balance representation

### PR DESCRIPTION
**Pull request by JoelJesudason**
Still need to add in GraphQL pull to get the user's current stream amount

One step at a time, let's go!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -553,20 +571,37 @@ class App extends Component {
...
-                    <td class="one-off">Your DAI Balance: <span id="balance-0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063">Loading</span></td>
+                    <td class="one-off">Your DAI Balance: {(this.state.tokenBalances[DAIAddress]/Math.pow(10,18)).toFixed(6)}</td>
                   </tr>
                 </table>
               </div>
             </div>
+            <br/>
+            <div class="card">
+              <div class="card-body">
+                <h5 class="card-title">Downgrade ETH to ETHx</h5>
```
Comment (mikeghen) [inline src/App.js:582]: ETHx to ETH you mean

File: src/App.js [hunk 2]
```diff
@@ -202,11 +203,12 @@ class App extends Component {
   async getTokenBalance(userAddress,tokenAddress) {
     var tokenInst = new this.state.web3.eth.Contract(erc20ABI,tokenAddress);
     tokenInst.methods.balanceOf(userAddress).call().then(function (bal) {
-        document.getElementById(`balance-${tokenAddress}`).innerHTML = (bal/1000000000000000000).toFixed(7)
+        // document.getElementById(`balance-${tokenAddress}`).innerHTML = (bal/1000000000000000000).toFixed(7)
```
Comment (mikeghen) [inline src/App.js:206]: why is this still here? lol

**Review by mikeghen — CHANGES_REQUESTED** [review]
This looks good, couple comments, can we just do downgrade DAIx and upgrade ETHx while we're at it?

File: src/App.js (hunk 2 above)
Comment (JoelJesudason) [reply src/App.js:206]: idk must have missed it. Didn't delete it by the time i pushed

File: src/App.js (hunk 1 above)
Comment (JoelJesudason) [reply src/App.js:582]: right will fix

**Pull request merged**